Lexdao-Constitutionv02.md

```diff
@@ -0,0 +1,160 @@
+
+# Article II - Membership
+
+**Section 2.1 - Eligibility** Membership as defined in Wyo. Stat. Title 17, Ch. 22 shall be voluntary and application open to any individual of at least 18 years of age whose purpose or presumed intent is to contribute, to build, and to use the services of LexDAO and is willing to accept the responsibilities and terms of membership. 
+
+**Section 2.2 - Admission.** The membership of LexDAO applicants will be granted or denied by vote upon the submission of required information on a prescribed application form, paying any required application fee, and contributing to LexDAO, if any, all as determined by the membership. The contribution requirement may be waived in whole or in part for persons of low income. LexDAO is a decentralized network of members and may admit or deny for any arbitrary purpose or lack of purpose.
+
+All LexDAO governing members will receive 1000 LexDAO Coop (each a “LDC”). LDC is a membership and voting token for participating in and improving the governance of LexDAO via the Designated Smart Contract. Once the admission requirements are met the prospective member may be put up to a vote of the full LDC membership. LexDAO will consistently review, and if necessary, LDC members can make adjustments to the admission requirements based on its evolving needs.  Current path to initiate a proposal to join membership and  to signal members for a vote can be found in the LexDAO Handbook, and may be updated from time to time.
+
+**Section 2.3 - Rights.** LexDAO Members shall utilize the “Designated Smart Contract” (a smart contract deployed to the Designated Blockchain at the Designated Blockchain Address (currently a MYSTIC application) as the exclusive method of holding, allocating among the LDC Members, and spending, or otherwise distributing any Tokens that are LexDAO property, of minting and issuing LDC Membership Tokens and holding and recording votes of the LDC Members. LexDAO may also utilize the Designated Smart Contract to administer and facilitate certain other arrangements and transactions involving LexDAO, the LDC Members and/or third parties.
+
+LexDAO Members’ rights and responsibilities are controlled by the use of the Designated Smart Contact used to conduct its governance and LexDAO activities. LexDAO's Aragon Agent is located at LexDAO’s designated Ethereum Web3 address **(0x97103fda00a2b47EaC669568063C00e65866a633)** through LexDAO governance to act on behalf of LexDAO, to contract on behalf of the DAO through a LDC membership vote.  LDC members will cast votes and carry out the decisions made on the Designated Smart Contract.  LDC tokens are held in Account Addresses, key paired wallets controlled by LDC Members in a designated hexadecimal address (“Member Web3 Account”). The registry of Member Web3 Account addresses holding LDC governance tokens of LexDAO can be found here: https://nightly.aragon.org/#/lexdao/0xc191dc522672e10441bb9ee7c58946e9a8ebe4c0/.
```

Comment:
The URL here to nightly aragon leads to a dead page.

Reply:
I believe this is from edit 1 of 5 changes to be committed and is no longer in the doc.
The final for committing is available at https://github.com/trtius/LexDAO-Constitution/blob/v02/LexDAO-Constitution%20v02.md